plaso/output/rawpy.py

```diff
@@ -62,15 +62,16 @@ def GetFormattedEvent(cls, event, event_data, event_tag):
             'value').format(attribute_name, event_data.data_type))
         attribute_value = ''
 
-      if attribute_name not in definitions.RESERVED_VARIABLE_NAMES:
-        attribute_string = '  {{{0!s}}} {1!s}'.format(
-            attribute_name, attribute_value)
-        additional_attributes.append(attribute_string)
+      if attribute_name == 'pathspec':
```

Comment:
It looks to me that the logic changes if you do this. Is this WAI?

Reply:
Not sure what you mean. It was `elif attribute_name != 'pathspec':` to exclude pathspec from the reserved_attributes results, but I've changed this to check for pathspec and move to the next attribute.